Link_sinFade_Library.ino

```diff
@@ -240,13 +240,22 @@ void HSVcheckLimits(float hsvArray[])  {
 //////////////////////////////////////////
 void HSVwriteToLEDs(float someHSV[], int someZone) {  // <<< Includes whiteBalance.
   float RGB_from_hsv[3];
+
   HSVehtoRGB(someHSV, RGB_from_hsv);              // Convert HSV to RGB on every iteration.
 
   //"White balance" is applied to the RGB form of the color HERE. Use RedScaler if needed.
   // colorRGB[0] = rgbf_color[0] * RedScaler / 100.0;  // RedScaler = 100   //  100.0
   RGB_from_hsv[1] = RGB_from_hsv[1] * GrnScaler / 100.0;  // GrnScaler = 77    //  67.0
   RGB_from_hsv[2] = RGB_from_hsv[2] * BluScaler / 100.0;  // BluScaler = 57    //  37.0
 
+
+  debug(F("B ")); debug(brightness()); debug(F(" "));
+  debug(F("@ ")); debug(someZone);
+  debug(F(" V ")); debug( someHSV[2] );
+  debug(F(" G ")); debug( RGB_from_hsv[1] ); // amplitude should track V
+  debugln();
```

Comment:
So, if debug is turned on, you get a bunch of data that makes it obvious that the brightness know works.